js/src/util.js

```diff
@@ -100,7 +67,7 @@ const Util = (($) => {
 
   const Util = {
 
-    TRANSITION_END: 'bsTransitionEnd',
+    TRANSITION_END: transition.end,
```

Comment:
Breaking change here, before that modification folks was able to listen to `bsTransitionEnd` event
Now they have to listen to their browser transition end event for example : `webkitTransitionEnd`